src/performance/osu/OsuScore.cpp

```diff
@@ -94,7 +94,7 @@ void OsuScore::computeAimValue(const Beatmap& beatmap)
 
 	// Longer maps are worth more
 	f32 LengthBonus = 0.95f + 0.4f * std::min(1.0f, static_cast<f32>(numTotalHits) / 2000.0f) +
-		(numTotalHits > 2000 ? log10(static_cast<f32>(numTotalHits) / 2000.0f) * 0.5f : 0.0f);
+					  (numTotalHits > 2000 ? log10(static_cast<f32>(numTotalHits) / 2000.0f) * 0.5f : 0.0f);
```

Comment:
tabs and spaces in one line. i'd cry if this was any other project.

Reply:
This is a result of VSCode auto format :| I can revert if you wish. I think this project should be converted to use spaces, because as you can see github is treating tabstops as 8 spaces here and ruining the alignment.

Reply:
I think it's fine. We eventually probably want to port this to .net anyways.